ledger/notary-plugin/src/main/java/net/corda/v5/ledger/notary/plugin/core/NotaryExceptionUnknown.java

```diff
@@ -6,8 +6,8 @@
 import org.jetbrains.annotations.Nullable;
 
 /**
- * The base class for notary exceptions that we cannot guarantee were fatal and might return with a different result
- * when retried. These type of exceptions will not invalidate the transaction immediately.
+ * The base class for notary exceptions that were fatal and might return with a different result when retried. These
```

Comment:
"that were fatal" - isn't that wrong as in they are not necessarily fatal to the tx? I find this change confusing.